dev-python/pipdeptree/pipdeptree-2.18.1.ebuild

```diff
@@ -39,9 +40,16 @@ PATCHES=(
 
 distutils_enable_tests pytest
 
+src_prepare() {
+	distutils-r1_src_prepare
+
+	find -name '*.py' -exec \
+		sed -i -e 's:pip[.]_vendor[.]::' {} + || die
```

Comment:
This affects `src/pipdeptree/_adapter.py` which is installed. It is necessary to revbump in order to fix all existing installations.

Reply:
Should I rename the ebuild or create a new (-r1) one along side the older one?